src/gurt/telemetry.c

```diff
@@ -2156,7 +2156,7 @@ d_tm_get_shared_memory(int srv_idx)
 
 	/** create a unique key for this instance */
 	key = D_TM_SHARED_MEMORY_KEY + srv_idx;
-	shmid = shmget(key, 0, 0666);
+	shmid = shmget(key, 0, 0);
```

Comment:
why 0?

Reply:
I reread the definition of shmget, and it appears that if the caller is not creating the shared memory segment then the shmflag is not used.  It doesn't appear to be harmful to supply the permissions again, but it didn't appear to be necessary either.